TestSwitchApi/TestSwitchApi/Models/Response/CandidateTestStatus.cs

```diff
@@ -0,0 +1,26 @@
+﻿using TestSwitchApi.Models.DataModels;
+
+namespace TestSwitchApi.Models.Response
+{
+    public class CandidateTestStatus
+    {
+        public string TestName { get; set; }
+        public string TestStatus { get; set; }
```

Comment:
This can just have type `Status`, and then you can also get rid of the `ToString()`s! :)

Reply:
changed to type status and added AddJsonOptions(opts => in StartUp.cs